src/plugins/analysis/strings/code/strings.py

```diff
@@ -20,25 +21,17 @@ class AnalysisPlugin(AnalysisBasePlugin):
         (b'[\x09-\x0d\x20-\x7e]{$len,}', 'utf-8'),
         (b'(?:[\x09-\x0d\x20-\x7e]\x00){$len,}', 'utf-16'),
     ]
-    FALLBACK_MIN_LENGTH = '8'
 
     def additional_setup(self):
         self.regexes = self._compile_regexes()
 
     def _compile_regexes(self) -> List[Tuple[Pattern[bytes], str]]:
-        min_length = self._get_min_length_from_config()
+        min_length = str(getattr(cfg, self.NAME, {}).get('min-length', 8))
```

Comment:
Ideally a plugin could register itself in the pydantic model but I don't think that this is possible.
`gettattr` is perfectly fine since that is exactly what we want to do: Get attributes that don't have their name defined in advance but rather at runtime.